CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Change log for PsDscResources
 
 ## Unreleased
+* Group
+  * Workaround added for the [Issue #82](https://github.com/PowerShell/PSDscResources/issues/82)
```

Comment:
Could this say that the work around is to catch the exception? It isn't 100% clear how the work around applies.

Reply:
Updated as per request